pkg/cmd/cli/restore/create.go

```diff
@@ -170,12 +179,63 @@ func (o *CreateOptions) Validate(c *cobra.Command, args []string, f client.Facto
 	return nil
 }
 
+func mostRecentBackup(backups []api.Backup, allowedPhases ...api.BackupPhase) *api.Backup {
+	// sort the backups in descending order of start time (i.e. most recent to least recent)
```

Comment:
Super nit pic: this comment should be the comment for the function.

Reply:
well, it doesn't fully describe the function, only what the sort does, but i'll add a godoc comment for the function

Reply:
Ahhh my bad.